index.d.ts

```diff
@@ -237,6 +237,14 @@ export interface CreateUserOptions {
 }
 
 /**
+ * The options object passed into the updateProfile function.
+ */
+export interface UpdateProfileOptions {
+    displayName: string;
```

Comment:
Yeah, that would be nice, because I'm currently using this project for a nativescript app and I'm sure I'll be contributing some more to this project during the app-development.